releases/release-4.0.3.md

```diff
@@ -0,0 +1,167 @@
+    - Fix dst store filter in hot Region scheduler [#2627](https://github.com/pingcap/pd/pull/2627)
+    - Fix the issue that the TSO request might take too long when PD leader switches [#2622](https://github.com/pingcap/pd/pull/2622)
+    - Fix the issue of stale scheduler after leader change [#2608](https://github.com/pingcap/pd/pull/2608)
+    - Fix the issue that when enabling placement rules, sometimes replicas of a Region might not be able to adjust to the best location [#2605](https://github.com/pingcap/pd/pull/2605)
+    - Fix the issue that the deployment path of the store is not updated if the deployment directory is moved [#2600](https://github.com/pingcap/pd/pull/2600)
+    - Prevent store limit from changing to zero [#2588](https://github.com/pingcap/pd/pull/2588)
+
++ TiDB Dashboard
+
+    - Fix TiDB connection error when TiDB is scaled in [#689](https://github.com/pingcap-incubator/tidb-dashboard/pull/689)
+    - Fix the issue that TiFlash instances are not displayed in the log searching page [#680](https://github.com/pingcap-incubator/tidb-dashboard/pull/680)
+    - Fix metrics selection reset after refresh in overview page [#663](https://github.com/pingcap-incubator/tidb-dashboard/pull/663)
+    - Fix connection issue in some TLS scenarios [#660](https://github.com/pingcap-incubator/tidb-dashboard/pull/660)
+    - Fix the issue that the language dropdown box is not displayed correctly in some cases [#677](https://github.com/pingcap-incubator/tidb-dashboard/pull/677)
+
++ TiFlash
+
+    - Fix the issue that TiFlash crashes after renaming the name of the primary key column [#916](https://github.com/pingcap/tics/pull/916)
+    - Fix the issue that concurrent learner read and remove Region might lead to deadlocks [#872](https://github.com/pingcap/tics/pull/872)
+
++ Tools
+
+    + TiCDC
+
+        - Fix the issue that TiCDC leaks memory in some cases [#704](https://github.com/pingcap/ticdc/pull/704)
+        - Fix the issue that unquoted table name causes the SQL syntax error [#676](https://github.com/pingcap/ticdc/pull/676)
+        - Fix the issue that the processor does not fully exit after `p.stop` is called [#693](https://github.com/pingcap/ticdc/pull/693)
+
+    + Backup & Restore (BR)
+
+        - Fix the issue that the backup time might be negative [#405](https://github.com/pingcap/br/pull/405)
+
+    + Dumpling
+
+        - Fix the issue that Dumpling omits the `NULL` value when `--r` is specified [#119](https://github.com/pingcap/dumpling/pull/119)
+        - Fix the bug that flush tables might not work for tables to dump [#117](https://github.com/pingcap/dumpling/pull/117)
+
+    + TiDB Lightning
+
+        - Fix the issue that `--log-file` does not take effect [#345](https://github.com/pingcap/tidb-lightning/pull/345)
+
+    + TiDB Binlog
+
+        - Fix the issue that not enable TLS to save checkpoint when enabling TLS for replicating [#988](https://github.com/pingcap/tidb-binlog/pull/988)
```

Comment:
@july2993 PTAL
```suggestion
        - Fix the issue that when TiDB Binlog replicates data to the downstream with TLS enabled, Drainer cannot be started which occurs because TLS is not enabled on the database used to store the checkpoint [#988](https://github.com/pingcap/tidb-binlog/pull/988)
```

Reply:
s/database used to store the checkpoint/database driver used to update the checkpoint/